cmd/qgb/query/cmd.go

```diff
@@ -0,0 +1,229 @@
+package query
+
+import (
+	"context"
+	"fmt"
+	"math/big"
+	"os"
+	"strconv"
+	"time"
+
+	celestiatypes "github.com/celestiaorg/celestia-app/x/qgb/types"
+	"github.com/celestiaorg/orchestrator-relayer/cmd/qgb/common"
+	"github.com/celestiaorg/orchestrator-relayer/p2p"
+	"github.com/celestiaorg/orchestrator-relayer/rpc"
+	"github.com/celestiaorg/orchestrator-relayer/types"
+	ds "github.com/ipfs/go-datastore"
+	dssync "github.com/ipfs/go-datastore/sync"
+	"github.com/libp2p/go-libp2p"
+	"github.com/libp2p/go-libp2p/core/peer"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	tmlog "github.com/tendermint/tendermint/libs/log"
+)
+
+func Command() *cobra.Command {
+	queryCmd := &cobra.Command{
+		Use:          "query",
+		Aliases:      []string{"q"},
+		Short:        "Query relevant information from a running QGB",
+		SilenceUsage: true,
+	}
+
+	queryCmd.AddCommand(
+		Signers(),
+	)
+
+	queryCmd.SetHelpCommand(&cobra.Command{})
+
+	return queryCmd
+}
+
+func Signers() *cobra.Command {
+	command := &cobra.Command{
+		Use:   "signers <nonce>",
+		Args:  cobra.ExactArgs(1),
+		Short: "Queries the QGB for attestations signers",
+		Long: "Queries the QGB for attestations signers. The nonce is the attestation nonce that the command" +
+			" will query signatures for. It should be either a specific nonce starting from 2 and on." +
+			" Or, use 'latest' as argument to check the latest attestation nonce",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			config, err := parseFlags(cmd)
+			if err != nil {
+				return err
+			}
+
+			// creating the logger
+			logger := tmlog.NewTMLogger(os.Stdout)
+			logger.Debug("initializing queriers")
+
+			ctx, cancel := context.WithCancel(cmd.Context())
+			defer cancel()
+
+			stopFuncs := make([]func() error, 0)
+			defer func() {
+				for _, f := range stopFuncs {
+					err := f()
+					if err != nil {
+						logger.Error(err.Error())
+					}
+				}
+			}()
```

Comment:
[not blocking]

ideally we just call this later and pass the slice after we have appended to it. This should work, I think its just not ideal. also, since we're already `make`ing a slice, then we might as well use the expected capacity

```go
stopFuncs := make([]func() error, 0, 1)
```